packages/integration-tests/src/api/connector.ts

```diff
@@ -2,13 +2,20 @@ import type { Connector, ConnectorResponse, CreateConnector } from '@logto/schem
 
 import { authedAdminApi } from './api.js';
 
+/**
+ * We are using `id` and `connectorId` here:
+ *
+ * - `id` is used to identify connectors from the database.
+ * - `connectorId` is used to identify connectors - more specifically, connector factories - in packages/connectors
+ * that contain metadata (considered connectors' FIXED properties) and code implementation (which determines how connectors work).
+ */
```

Comment:
Might need a better name? Is it widely used cross our connector code base that id stands for connector instances id in DB and connector stands for the factory?  A little bit confusing.  cc: @logto-io/eng

Reply:
In the IT, we can rename `connectorId` as `connectorFactoryId` and we can update the schema when we are going to boost to logto v2?

Reply:
Added LOG-6104 to track this update.